src/OrderFormAcceptanceTests.Steps/Steps/CompleteOrder.cs

```diff
@@ -24,8 +25,10 @@ public void WhenTheUserChoosesToCompleteTheOrder()
         }
 
         [StepDefinition(@"the User confirms to complete the Order")]
-        public void WhenTheUserConfirmsToCompleteTheOrder()
+        public async System.Threading.Tasks.Task WhenTheUserConfirmsToCompleteTheOrderAsync()
```

Comment:
Make use of a using statement at the top of the file, so don't need to duplicate `System.Threading.Tasks.` e.g. `using System.Threading.Tasks;`